### DISCUSSION TITLE
Forms: User preference ranking

### DISCUSSION THREAD

**Discussion by Wulfheart** [filamentphp/filament#5949] [Help]
Hello,

I have a list of selection from which a user has the ability to rank them based on preference. I would like the user to be able to drag and drop these choice in the preferred order of choice. Is this somehow possible? I thought of a repeater but a repeater has multiple fields and I only need some text and it looks clunky for this use case to me.
I would also be fine to use a plug-in.

Thank you in advance for your help!

**Comment by danharrin** [comment]
Maybe https://filamentphp.com/plugins/simple-repeater
